pkg/controller/application/reconciliation/application_reconciler.go

```diff
@@ -16,10 +19,15 @@ const applicationReconcilerIdentifier = "kubernetes manifests"
 // applicationReconciler contains service and metadata for the relevant resource
 type applicationReconciler struct {
 	client client.ApplicationService
+	logger logging.Logger
 }
 
 // Reconcile knows how to do what is necessary to ensure the desired state is achieved
 func (a *applicationReconciler) Reconcile(ctx context.Context, meta reconciliation.Metadata, state *clientCore.StateHandlers) (reconciliation.Result, error) {
+	a.logger = logging.GetLogger("application", "reconcile")
```

Comment:
I would initialize the logger in the New function - but this was maybe just a test?